assets-src/styles/sass/30-base/_tables.scss

```diff
@@ -7,6 +7,8 @@
 table {
     border: 1px solid $border-color;
     border-collapse: collapse;
+	margin-bottom: 1rem;
+	margin-top: 1rem;
     width: 100%;
```

Comment:
Code indentation needs fixing.

Reply:
Please re-check after pulling latest commits. I'm not sure if my default IDE indent spacing has changed between spaces and tabs.